### PR TITLE
Fixed link in cac-consult-rubric

### PR DESCRIPTION
**Pull request by martinosorb**
Link appears to be broken.

### CODE REVIEW

**Comment by maneesha** [comment]
Thanks @martinosorb 

Checking with @tobyhodges -- the page linked to in this PR goes to an overview of the CAC: https://docs.carpentries.org/topic_folders/lesson_development/curriculum_advisory_committees.html

The first link on that page goes to the consultation rubric: https://docs.carpentries.org/topic_folders/lesson_development/cac-consult-rubric.html

Which would you like the link in this PR to point to?

**Comment by martinosorb** [comment]
The consultation rubric is the very page this link is on, so it should be the other one.

File: topic_folders/lesson_development/cac-consult-rubric.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Curriculum Advisory Committee Consultation Rubric
 
 This rubric defines the division of responsibilities between The Carpentries Maintainers and The Carpentries Curriculum Advisory Committees (CACs). 
-[Roles and responsibilities](https://docs.carpentries.org/topic_folders/lesson_development/lesson_development_roles.html#curriculum-advisory-committees) 
+[Roles and responsibilities](https://docs.carpentries.org/topic_folders/lesson_development/curriculum_advisory_committees.html)
```
Comment (tobyhodges) [inline topic_folders/lesson_development/cac-consult-rubric.md:4]:
```suggestion
[Roles and responsibilities](https://docs.carpentries.org/topic_folders/lesson_development/curriculum_advisory_committees.html#roles-and-responsibilities) 
```

I think the link could be further improved by taking the visitor directly to the relevant section on roles and responsibilities of CACs

**Review by tobyhodges — COMMENTED** [review]
I agree with @martinosorb, and approve of this change. However, I have suggested linking to the specific section of the handbook page where CAC roles and responsibilities are described.

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**